tests/publiccloud/img_proof.pm

```diff
@@ -125,6 +125,18 @@ sub run {
     );
 
     # Because the IP address of instance might change during img_proof due to the hard-reboot, we need to re-add the ssh public keys
+    my $ssh_dir = "~/.ssh";
+    if (script_run("test -d $ssh_dir") != 0) {
+        assert_script_run("mkdir -p $ssh_dir");
+        assert_script_run("chmod 700 $ssh_dir");
+    }
+
+    my $known_hosts_file = "$ssh_dir/known_hosts";
+    if (script_run("test -f $known_hosts_file") != 0) {
+        assert_script_run("touch $known_hosts_file");
```

Comment:
I think we can do this `touch` unconditionally and don't care about the permissions since it doesn't hold sensitive info anyway.

Reply:
we can not ignore permissions . openssh will refuse to use a key with improper permissions

Reply:
This is not a key.

Reply:
ah right , then yeah make sense !